task-list/app/models/person.rb

```diff
@@ -0,0 +1,10 @@
+class Person < ActiveRecord::Base
+  has_many :tasks
+
+  def count_incomplete
+    t = self.tasks
+    incomplete = t.where(completed_date: nil)
+    incomplete.length
```

Comment:
Generally we should prefer to use `count` on ActiveRecord relations (the array-link objects that are returned by `all` and `where`), because they "compose" together better and allow ActiveRecord to use more efficient SQL code.

If you try it out in rails console vs. using `length` you should be able to see how the generated SQL code is different.